doc/npc_behaviour.asc

```diff
@@ -79,7 +79,7 @@ struct tr_box   // 20 bytes
     uint32_t Xmin;
     uint32_t Xmax;
      int16_t TrueFloor;     // Height value in global units
-     int16_t OverlapIndex;  // Index into Overlaps[].
+     int16_t OverlapIndex;  // bit 0..14 is the index into Overlaps[], bit 15 is unknown (may occur behind a door)
```

Comment:
Could you change the type to `uint16_t` to make sure that using `abs()` is the wrong way of masking bit 15 out?